pentaho-i18n-bundle/src/main/java/org/pentaho/osgi/i18n/LocalizationService.java

```diff
@@ -17,15 +17,21 @@
 
 package org.pentaho.osgi.i18n;
 
+import org.osgi.framework.Bundle;
+
 import java.util.List;
 import java.util.Locale;
 import java.util.ResourceBundle;
 import java.util.regex.Pattern;
 
-/**
- * Created by bryan on 9/5/14.
- */
 public interface LocalizationService {
-  public ResourceBundle getResourceBundle( String name, Locale locale );
-  public List<ResourceBundle> getResourceBundles( Pattern keyRegex, Locale locale );
+  String CAPABILITY_NAMESPACE = "org.pentaho.webpackage";
+
+  ResourceBundle getResourceBundle( Bundle bundle, String key, Locale locale );
```

Comment:
We probably want to have separate interfaces for OSGI and non-OSGI. So the ```getResourceBundle( org.osgi-framework.Bundle ...)``` would be in a different interface. Let me think a bit about this before getting back to you.